### ISSUE TITLE
Panic after ICMP probe error

### ISSUE THREAD

**Issue by grobie** [prometheus/blackbox_exporter#121]
```
ERRO[0247] Error resolving address mp3skull.com: lookup mp3skull.com on 8.8.8.8:53: no such host  source=icmp.go:81
2017/03/04 01:57:19 http: panic serving 127.0.0.1:58324: runtime error: invalid memory address or nil pointer dereference
goroutine 192922 [running]:
net/http.(*conn).serve.func1(0xc420843680)
        /usr/local/go/src/net/http/server.go:1491 +0x12a
panic(0x83a940, 0xc42000a0c0)
        /usr/local/go/src/runtime/panic.go:458 +0x243
main.probeICMP(0xc424cb975e, 0xc, 0xab3000, 0xc423c86000, 0xc4200de4d8, 0x4, 0x3b9aca00, 0x0, 0x0, 0x0, ...)
        /go/src/github.com/prometheus/blackbox_exporter/icmp.go:84 +0x1fc
main.probeHandler(0xab3000, 0xc423c86000, 0xc4208c0e10, 0xc4200240a0)
        /go/src/github.com/prometheus/blackbox_exporter/main.go:130 +0x218
main.main.func1(0xab3000, 0xc423c86000, 0xc4208c0e10)
        /go/src/github.com/prometheus/blackbox_exporter/main.go:170 +0x4a
net/http.HandlerFunc.ServeHTTP(0xc4200de9c0, 0xab3000, 0xc423c86000, 0xc4208c0e10)
        /usr/local/go/src/net/http/server.go:1726 +0x44
net/http.(*ServeMux).ServeHTTP(0xad3ae0, 0xab3000, 0xc423c86000, 0xc4208c0e10)
        /usr/local/go/src/net/http/server.go:2022 +0x7f
net/http.serverHandler.ServeHTTP(0xc420014780, 0xab3000, 0xc423c86000, 0xc4208c0e10)
        /usr/local/go/src/net/http/server.go:2202 +0x7d
net/http.(*conn).serve(0xc420843680, 0xab3980, 0xc424cb9700)
        /usr/local/go/src/net/http/server.go:1579 +0x4b7
created by net/http.(*Server).Serve
        /usr/local/go/src/net/http/server.go:2293 +0x44d
```

**Comment by grobie** [comment]
Hm, I can't reproduce this issue in isolation. I was load testing the blackbox_exporter and got such panics quite regularly, but only for ICMP probes (sending each 1000x ICMP, HTTP and TCP probes).

**Comment by grobie** [comment]
Alright, this has been fixed in #106 already.
